common/templates/xblock_v2/xblock_iframe.html

```diff
@@ -215,6 +215,13 @@
                 }
                 return url;
             },
+            notify: () => {
+              /** 
+               * Used in `studio_view`
+               * 
+               * TODO: Do nothing for now
```

Comment:
I'm not sure what to put here yet, but it could be implemented in a separate task after the prototype is approved.